Build failing

@jimbobbennett 

Hey Jim! The build is failing for this repo. Not sure if that's expected (or if I did something to mess it up?)
The error it gives is about permissions so I think it's a config problem.

https://github.com/microsoft/AcademicContent/runs/1200985370

Run git config --global user.email "caat@microsoft.com"
./deploy.sh: 3: [: microsoft: unexpected operator

> academic-repo@1.0.0 docs:build /home/runner/work/AcademicContent/AcademicContent
> vuepress build docs

wait Extracting site metadata...
tip Apply theme @vuepress/theme-default ...
tip Apply plugin container (i.e. "vuepress-plugin-container") ...
tip Apply plugin @vuepress/last-updated (i.e. "@vuepress/plugin-last-updated") ...
tip Apply plugin @vuepress/register-components (i.e. "@vuepress/plugin-register-components") ...
tip Apply plugin @vuepress/active-header-links (i.e. "@vuepress/plugin-active-header-links") ...
tip Apply plugin @vuepress/search (i.e. "@vuepress/plugin-search") ...
tip Apply plugin @vuepress/nprogress (i.e. "@vuepress/plugin-nprogress") ...
[info] [webpackbar] Compiling Client
[info] [webpackbar] Compiling Server
Browserslist: caniuse-lite is outdated. Please run the following command: `npx browserslist --update-db`
[success] [webpackbar] Server: Compiled successfully in 11.83s
[success] [webpackbar] Client: Compiled successfully in 11.93s
wait Rendering static HTML...
success Generated static files in docs/.vuepress/dist.

Initialized empty Git repository in /home/runner/work/AcademicContent/AcademicContent/docs/.vuepress/dist/.git/
[master (root-commit) 336d562] deploy
 21 files changed, 235 insertions(+)
 create mode 100644 CONTRIBUTING.html
 create mode 100644 assets/css/0.styles.47b177c7.css
 create mode 100644 assets/img/search.83621669.svg
 create mode 100644 assets/js/10.92811617.js
 create mode 100644 assets/js/11.020a55c7.js
 create mode 100644 assets/js/2.66562e48.js
 create mode 100644 assets/js/3.7a41b788.js
 create mode 100644 assets/js/4.5f60d4ea.js
 create mode 100644 assets/js/5.03ea7ffc.js
 create mode 100644 assets/js/6.c5c58b40.js
 create mode 100644 assets/js/7.7cbd9f73.js
 create mode 100644 assets/js/8.7f58d49d.js
 create mode 100644 assets/js/9.6274317f.js
 create mode 100644 assets/js/app.5747f694.js
 create mode 100644 assets/logo.png
 create mode 100644 content/event-content.html
 create mode 100644 content/hackathon-content.html
 create mode 100644 content/k-12-content.html
 create mode 100644 content/learning-content.html
 create mode 100644 favicon.ico
 create mode 100644 index.html
Warning: Permanently added the RSA host key for IP address '140.82.112.3' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Error: Process completed with exit code 128.

--- a/.github/workflows/DeployToGitHubPages.yml
+++ b/.github/workflows/DeployToGitHubPages.yml
@@ -23,7 +23,7 @@ jobs:
       run: npm install
     - name: 'Deploy to GitHub pages'
       run: |
-        ./deploy.sh microsoft
+        ./deploy.sh
     - name: Deploy
       uses: peaceiris/actions-gh-pages@v3
       with:

--- a/.github/workflows/DeployToGitHubPages.yml
+++ b/.github/workflows/DeployToGitHubPages.yml
@@ -23,6 +23,11 @@ jobs:
       run: npm install
     - name: 'Deploy to GitHub pages'
       run: |
-        git config --global user.email "caat@microsoft.com"
-        git config --global user.name "Bit"
-        ./deploy.sh microsoft microsoft ${{ secrets.GITHUB_TOKEN }}
+        ./deploy.sh microsoft
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./docs/.vuepress/dist
+        user_name: 'Bit'
+        user_email: 'caat@microsoft.com'

--- a/.github/workflows/DeployToGitHubPages.yml
+++ b/.github/workflows/DeployToGitHubPages.yml
@@ -1,4 +1,5 @@
 # When something is pushed to master, rebuild the docs
+name: DeployToGitHubPages
 
 on:
   push:

--- a/deploy.sh
+++ b/deploy.sh
@@ -1,10 +1,10 @@
 #!/usr/bin/env sh
 
-if [ "$1" == "" ]; then
-    echo "Usage: ./deploy.sh <org>"
-    echo "Where <org> is the name of the GitHub org to push to"
-    exit -1
-fi
+# if [ "$1" == "" ]; then
+#     echo "Usage: ./deploy.sh <org>"
+#     echo "Where <org> is the name of the GitHub org to push to"
+#     exit -1
+# fi
 
 # abort on errors
 set -e
@@ -16,19 +16,19 @@ cp ./CONTRIBUTING.md ./docs
 npm run docs:build
 
 # navigate into the build output directory
-cd docs/.vuepress/dist
+# cd docs/.vuepress/dist
 
 # if you are deploying to a custom domain
 # echo 'www.example.com' > CNAME
 
-git init
-git add -A
-git commit -m 'deploy'
+# git init
+# git add -A
+# git commit -m 'deploy'
 
 # if you are deploying to https://<USERNAME>.github.io
 # git push -f git@github.com:<USERNAME>/<USERNAME>.github.io.git master
 
 # if you are deploying to https://<USERNAME>.github.io/<REPO>
-git push -f git@github.com:$1/computerscience.git master:gh-pages
+# git push -f git@github.com:$1/computerscience.git master:gh-pages
 
-cd -
+# cd -

